winbuild/build_prepare.py

```diff
@@ -293,12 +293,8 @@ def cmd_msbuild(
         },
         "build": [
             cmd_rmdir("objs"),
-            cmd_msbuild(
-                r"builds\windows\vc2010\freetype.sln", "Release Static", "Clean"
-            ),
-            cmd_msbuild(
-                r"builds\windows\vc2010\freetype.sln", "Release Static", "Build"
-            ),
+            cmd_msbuild("MSBuild.sln", "Release Static", "Clean"),
+            cmd_msbuild("MSBuild.sln", "Release Static", "Build"),
```

Comment:
This change means that the patches above are no longer being used. You can see in the test results that the CBDT/SBIX/WOFF2 tests are being skipped.

Reply:
I've created #8403 to fix this.